components/select_team/select_team.jsx

```diff
@@ -292,7 +316,10 @@ export default class SelectTeam extends React.Component {
                 <AnnouncementBar/>
                 {headerButton}
                 <div className='col-sm-12'>
-                    <div className={'signup-team__container'}>
+                    <div
+                        className={'signup-team__container'}
+                        style={{maxWidth: '800px'}}
```

Comment:
Was this an intended change?

Reply:
@sudheerDev yes. I didn't want to modify the `signup-team__container` class because it's used in multiple places so I thought adding a simple inline style would do the trick. Do you suggest move this out to a class?

Reply:
@sudheerDev any follow-up views on this?

Reply:
i am ok with `inline-style` actually i prefer it if it is just one style and a non resuable css class. There is no point in making a css class here. 

I thought this was unintentionally added hence the question in the first place